### PR TITLE
Afrikaans Quotes

### PR DESCRIPTION
**Pull request by Jeff0502**
### Description

Closes #

### CODE REVIEW

**Comment by Miodec** [comment]
Please add a few more, and also translate them so i can verify their content.

**Comment by Jeff0502** [comment]
Sure! Will 7 (10 in total) more be enough?

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4699784170

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4699834327

**Comment by Jeff0502** [comment]
{
       "text": "His eyes float like a boat on the sea, 'Dude, I think I'm a cyborg now.' Only then do I understand. Plank implanted the transponder in his hand.",
       "source": "Dark Web, Fanie Viljoen",
       "length": 147,
       "id": 1
     },
     {
       "text": "Yes, the whole world falls silent and listens in the dark hour, to the night sounds of Lisa's piano.",
       "source": "Lisa's piano - Koos Kombuis",
       "length": 98,
       "id": 2
     },
     {
       "text": "But I stood my ground and said I was going, man-alone one dark night. I steeled myself and sent for my horse, but I never expected such a thing.",
       "source": "Op Path Na Nooitgedacht - Koos du Plessis",
       "length": 144,
       "id": 3
     },
     {
       "text": "'If Google or some other app asks you for access to the personal details on your mobile phone, you decide with that one simple click of Accept: I hereby surrender my autonomy to the Technium. I hereby initiate myself surrender to it voluntarily.'",
       "source": "Dark Web, Fanie Viljoen",
       "length": 255,
       "id": 4
     },
     {
       "text": "Life hit your mom and life hit your dad, maybe you shouldn't hit them alongside ",
       "source": "Lien se Lankstaanskoene, Derick van der Walt",
       "length": 91,
       "id": 5
     },
     {
       "text": "That's what fear feels like: Fire flashes through your body, every nerve jolts awake, adrenaline throws your mind into overdrive, your eyes snap open, your breathing stops and your chest squeezes. And you hear it - your own heart. Beat. Beat. Beat.",
       "source": "Underworld, Fanie Viljoen",
       "length": 230,
       "id": 6
     },
     {
       "text": "I know an old, old song of life and ways, of long-gone ships, the cellars of the sea. The words are forgotten, and yet the tune rolls like vaguely familiar bars in a very old story.",
       "source": "Children of the wind - Koos du Plessis",
       "length": 196,
       "id": 7
     },
     {
       "text": "Forget to forget me. Remember to remember. And know that you always know, I love you.",
       "source": "Forget Om Te Forget (Remember To Remember Me) - Anton Goosen",
       "length": 93,
       "id": 8
     }

**Comment by Miodec** [comment]
> {
> "text": "Life hit your mom and life hit your dad, maybe you shouldn't hit them alongside ",
> "source": "Lien se Lankstaanskoene, Derick van der Walt",
> "length": 91,
> "id": 5
> },

I would say remove this one

File: frontend/static/languages/afrikaans_10k.json
Comment (Jeff0502) [inline frontend/static/languages/afrikaans_10k.json:1]: "sex",
    "sexuality",
    "sexual",
    "sexual",
"rape",
    "raped",

Comment (Jeff0502) [reply frontend/static/languages/afrikaans_10k.json:1]: Edit: Explicit, not obscene/offensive

**Review by Miodec — APPROVED** [review]

**Pull request merged**